Gruntfile.js

```diff
@@ -1,63 +1,187 @@
-'use strict';
+var request = require('superagent');
 
 module.exports = function(grunt) {
+  'use strict';
+
+  require('load-grunt-tasks')(grunt);
+
   grunt.initConfig({
-    'git-describe': {
-      options: {
+    pkg: grunt.file.readJSON('package.json'),
+    changelog: 'CHANGELOG.md',
+
+    'gh-release': {},
 
-      },
-      'run': {}
-    },
     closurecompiler: {
       minify: {
         requiresConfig: 'git-revision',
         files: {
-          "gen/axs_testing.js": [
```

Comment:
I'm not too fussed at this stage - the consistency in the new version is nice.

I'm not sure what tooling we have that integrates with GitHub, but I can look into it - would be nice to have some kind of hook to check this stuff rather than having to go back and forth in reviews.